Postman/Nfield Public API.postman_collection.json

```diff
@@ -0,0 +1,12332 @@
+												}
+											}
+										},
+										"url": {
+											"raw": "{{origin}}/v1/Surveys/{{SurveyId}}/InterviewInteractionsSettings",
+											"host": [
+												"{{origin}}"
+											],
+											"path": [
+												"v1",
+												"Surveys",
+												"{{SurveyId}}",
+												"InterviewInteractionsSettings"
+											]
+										},
+										"description": "(Online) Update interview interactions settings for the survey."
+									},
+									"response": []
+								}
+							]
+						},
+						{
+							"name": "Languages (Deprecated)",
+							"item": [
+								{
+									"name": "Get Languages",
+									"event": [
+										{
+											"listen": "test",
+											"script": {
+												"exec": [
+													"pm.test(\"Status code is 200\", function () {\r",
+													"    pm.response.to.have.status(200);\r",
+													"});\r",
+													"\r",
+													"pm.test(\"Status code name has string\", function () {\r",
+													"    pm.response.to.have.status(\"OK\");\r",
+													"});\r",
+													"\r",
+													"pm.test(\"Response contains LanguageId\", function () {\r",
+													"    var languages = pm.response.json();\r",
+													"});"
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"method": "GET",
+										"header": [
+											{
+												"key": "Authorization",
+												"value": "Basic {{AuthenticationToken}}",
+												"type": "text"
+											}
+										],
+										"url": {
+											"raw": "{{origin}}/v1/Surveys/{{SurveyId}}/Languages/",
+											"host": [
+												"{{origin}}"
+											],
+											"path": [
+												"v1",
+												"Surveys",
+												"{{SurveyId}}",
+												"Languages",
+												""
+											]
+										},
+										"description": "This method retrieves a list of languages defined for the survey with the specified surveyId. This list can be filtered and sorted using standard OData syntax."
+									},
+									"response": []
+								},
+								{
+									"name": "Get Language",
+									"event": [
+										{
+											"listen": "test",
+											"script": {
+												"exec": [
+													"pm.test(\"Status code is 200\", function () {\r",
+													"    pm.response.to.have.status(200);\r",
+													"});\r",
+													"\r",
+													"pm.test(\"Status code name has string\", function () {\r",
+													"    pm.response.to.have.status(\"OK\");\r",
+													"});\r",
+													"\r",
+													"pm.test(\"Response contains Id and Name\", function () {\r",
+													"    var language = pm.response.json();\r",
+													"    pm.expect(language.Id !== undefined).to.be.true;\r",
+													"    pm.expect(language.Name !== undefined).to.be.true;\r",
+													"});"
+												],
+												"t
```

Comment:
```suggestion
										"description": "Returns the counts for the specified survey. For all surveys, it will return the topline counts. For surveys with Quota, it will also include detailed counts per quota level"
```